org-code-javabuilder/lib/src/main/java/org/code/javabuilder/AWSFileManager.java

```diff
@@ -101,6 +105,17 @@ public URL getFileUrl(String filename) throws FileNotFoundException {
     }
   }
 
+  @Override
+  public void cleanUpTempDirectory(File tempFolder) throws IOException {
+    // Delete any leftover contents of the tmp folder from previous lambda invocations
+    // We can ignore the tempFolder since we are clearing the entire directory
+    Path toClear = Paths.get(System.getProperty("java.io.tmpdir"));
+    LoggerUtils.sendDiskSpaceLogs();
```

Comment:
nit: can we move the logs into the recursivelyClearDirectory method so we don't need to copy/paste the pattern a bunch of times? It will also make it easier to adjust the logs in the future.

Reply:
Yep will do